package.json

```diff
@@ -20,8 +21,8 @@
     "start": "npm run storybook",
     "storybook": "STORYBOOK_ENV=development storybook dev -p 9010",
     "storybook:clean": "rm -rf ./storybook-static",
-    "storybook:build": "npm run storybook:clean && STORYBOOK_ENV=production storybook build && touch ./storybook-static/.nojekyll",
-    "storybook:build:docs-only": "STORYBOOK_ENV=production storybook build --docs",
+    "storybook:build": "npm run build:documentation:functions && npm run storybook:clean && STORYBOOK_ENV=production storybook build && touch ./storybook-static/.nojekyll",
```

Comment:
Will this grow hideously?

Reply:
haha if we keep getting new processes that have to be run as part of the build, maybe. Hopefully won't be any more.